ibis/__init__.py

```diff
@@ -23,6 +17,8 @@
 
 def __dir__() -> list[str]:
     """Adds tab completion for ibis backends to the top-level module."""
+    from ibis import util
```

Comment:
What's in `ibis.util` that needs to be delayed loading? AFAICT pandas is only imported there if type checking?

Reply:
Nothing, this can be reverted.

Reply:
Done.